### DISCUSSION TITLE
Turborepo Support WebAssembly?

### DISCUSSION THREAD

**Discussion by litianming1** [vercel/turborepo#4046] [Ideas]
Turborepo Support WebAssembly,norm, compatibility Wait?
How to use it to load the WebAssembly module into the project. Efficient, portable, easy to use?

**Comment by anthonyshew** [comment]
Turborepo wouldn't be applicable to Wasm loading. Going to figure you meant Turbopack!

Looks like this pull request is good news for Wasm!: https://github.com/vercel/turbo/issues/4795
